### PR TITLE
Add timeout to requests

### PR DESCRIPTION
**Pull request by JeanMarcGoepfert**
### Overview

Adds a configurable timeout to get, put and delete requests which defaults to 15 seconds

### CODE REVIEW

File: src/Client.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ export default class Client {
   constructor(opts={}) {
     this.baseEndpointURL = opts.baseEndpointURL || "https://api.flippa.com/v3";
     this.accessToken = opts.accessToken || null;
+    this.timeout = opts.accessToken || 15000;
```
Comment (ags) [inline src/Client.js]: Should this be `opts.timeout`?

Comment (JeanMarcGoepfert) [inline src/Client.js]: Yep!

**Comment by ags** [comment]
:+1:

**Comment by bentheax** [comment]
:+1:

**Pull request merged**